### ISSUE TITLE
flags from terminal not set on ios update

### ISSUE THREAD

**Issue by madrazo** [haxenme/nme#407]
When doing from terminal
`nme update project.xml ios -Dmyflag -toolkit`

myflag is not "set" on project.xml.

Other targets that do not use "update" have the same behavior as writing
 `< set name="myflag" />`
on the xml

**Comment by madrazo** [comment]
I think that the "-toolkit" flag is ignored as well, since you need the precompiled nme lib.

**Comment by hughsando** [comment]
Yes, the problem here is that it does not store these values anywhere.
When you build from xcode, it runs nme again in xcode mode.  I need to make this command that it runs also include the original flags passed on the command line.
